templates/endpoint_policies/s3_org_only_access_plus_starport.tpl

```diff
@@ -0,0 +1,29 @@
+{
+  "Version": "2012-10-17",
+  "Statement": [
+    {
+      "Sid": "S3AccessWithinOrgOnly",
```

Comment:
May be put `org` out in full for the `Sid` just to make clear, ie `S3AccessWithinOrganisationOnly`